src/chrome/content/rules/dotabuff.com.xml

```diff
@@ -0,0 +1,124 @@
+<!--
+www.8t.dotabuff.com ¹
+www.9it.dotabuff.com ¹
+www.de.dotabuff.com ¹
+www.dotamax.dotabuff.com ¹
+www.en.dotabuff.com ¹
+2.en.dotabuff.com ¹
+5.en.dotabuff.com ¹
+oehardgamewww.en.dotabuff.com ¹
+terriblereality.en.dotabuff.com ¹
+www.fr.dotabuff.com ¹
+www.i8t.dotabuff.com ¹
+www.i9t.dotabuff.com ¹
+www.if.dotabuff.com ¹
+www.iit.dotabuff.com ¹
+www.iot.dotabuff.com ¹
+www.it.dotabuff.com ¹
+www.itf.dotabuff.com ¹
+www.itg.dotabuff.com ¹
+www.ith.dotabuff.com ¹
+www.itm.dotabuff.com ¹
+www.itt.dotabuff.com ¹
+www.iut.dotabuff.com ¹
+www.iy.dotabuff.com ¹
+www.jit.dotabuff.com ¹
+www.ka.dotabuff.com ¹
+www.kit.dotabuff.com ¹
+web.mail.dotabuff.com ¹
+www.pt.dotabuff.com ¹
+browse.pt.dotabuff.com ¹
+www.ru.dotabuff.com ¹
+2.ru.dotabuff.com ¹
+5.ru.dotabuff.com ¹
+en.ru.dotabuff.com ¹
+enwww.ru.dotabuff.com ¹
+ww2.ru.dotabuff.com ¹
+ww35.ru.dotabuff.com ¹
+ru.www.ru.dotabuff.com ¹
+www35.ru.dotabuff.com ¹
+status.dotabuff.com ¹
+www.t.dotabuff.com ¹
+pt.t.dotabuff.com ¹
+www.uit.dotabuff.com ¹
+www.ut.dotabuff.com ¹
+ww.w.dotabuff.com ¹
+www.webmail.dotabuff.com ¹
+www.www.dotabuff.com ¹
+2.www.dotabuff.com ¹
+bg.www.dotabuff.com ¹
+en.www.dotabuff.com ¹
+es.www.dotabuff.com ¹
+w.gowww.www.dotabuff.com ¹
+ru.www.dotabuff.com ¹
+zh.www.dotabuff.com ¹
+www.zh.dotabuff.com ¹
```

Comment:
> 4th+ level domain will mismatch anyway
Why would you sure about that? Even on same IP SNI can differ certificates for different domains.